src/app/GlobalStats/GlobalStats.tsx

```diff
@@ -99,24 +100,28 @@ const GlobalStats: React.FunctionComponent<any> = props => {
       return (
         <Card>
           <CardBody>
-            <Alert title={error} variant={AlertVariant.danger} isInline />}
+            <TableErrorState error={error} />
           </CardBody>
         </Card>
       );
     }
 
     if (!loading && !error && !stats.statistics_enabled) {
       return (
-        <EmptyState variant={EmptyStateVariant.full}>
-          <EmptyStateIcon icon={CubesIcon} />
-          <Title headingLevel="h5" size="lg">
-            Statistics are not enabled
-          </Title>
-          <EmptyStateBody>
-            Statistics are not enabled. To activate statistics, set
-            statistics=true in the configuration.
-          </EmptyStateBody>
-        </EmptyState>
+        <Card>
+          <CardBody>
+            <EmptyState variant={EmptyStateVariant.full}>
+              <EmptyStateIcon icon={CubesIcon} />
+              <Title headingLevel="h5" size="lg">
+                Statistics are not enabled
+              </Title>
+              <EmptyStateBody>
+                Statistics are not enabled. To activate statistics, set
```

Comment:
suggestion: `Statistic disabled. To enable statistics, ...`

Reply:
ok